converter/internal/common/common.go

```diff
@@ -59,6 +59,21 @@ func GetUniqueLabel(label string, currentCount int) string {
 	return fmt.Sprintf("%s_%d", label, currentCount)
 }
 
+// GetLabelWithPrefix creates a label with an optional labelPrefix and based on
+// the current count.
+func GetLabelWithPrefix(labelPrefix string, label string, currentCount int) string {
+	finalLabel := label
+	if currentCount != 0 {
+		finalLabel += fmt.Sprintf("_%d", currentCount+1)
+	}
+
+	if labelPrefix != "" {
+		finalLabel = labelPrefix + "_" + finalLabel
+	}
+
+	return finalLabel
+}
```

Comment:
I was thinking maybe we do something like this: https://go.dev/play/p/SleUy3-VQEA?v=
LMK what you think. I wanted to:
* support more than two string components in the label
* avoid calling with hard-coded `currentCount` set to 0

Reply:
I updated the implementation similar to what you suggested with a tweak for indexing and empty prefixes. The pattern is now 1 indexed without an index on the first label:

LABEL
LABEL_2
LABEL_3
etc

Some alternative patterns... I don't feel super strongly pro/against any honestly, but we can easily change it in the 1 place and update test outputs now:

LABEL
LABEL_1
LABEL_2
etc

OR

LABEL_0
LABEL_1
LABEL_2
etc

OR 

LABEL_1
LABEL_2
LABEL_3